client/src/pages/Message/MessageWindow/Header/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, Grid, Typography } from '@material-ui/core';
+import { Conversation } from '../../../../interface/Message';
+import AvatarIcon from '../../SideBar/AvatarIcon/AvatarIcon';
+import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
+import useStyles from './useStyles';
+
+export interface HeaderProps {
+  header?: Conversation;
+}
+export default function Header({ header }: HeaderProps): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Box className={classes.root}>
```

Comment:
The MUI Box component has flex box built into it, you don't need to add it using a class :)

Reply:
added CSS property inside the Box component.